src/lib/hooks/useDeleteAccountDialog.js

```diff
@@ -43,7 +44,7 @@ export default ({ API, showErrorDialog, store, theme }) =>
           {
             text: 'Delete',
             color: theme.colors.red,
-            onPress: async () => {
+            onPress: useOnPress(async () => {
```

Comment:
please move this outside useCallback

e.g. `const deleteHandler = useOnPress(async() { // ...`
then in useCallback / showErrorDialog - `onPress: deleteHandler`